guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,52 @@
+[id="installing-fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Installing fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Installing fapolicyd on {SmartProxyServer}
+endif::[]
+
+ifeval::["{context}" == "{project-context}"]
+You can install `fapolicyd` along with {ProjectServer} or can be installed on an existing {ProjectServer}.
+If you are installing fapolicyd along with the new {ProjectServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {Project} rules automatically.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+You can install `fapolicyd` along with {SmartProxyServer} or can be installed on an existing {SmartProxyServer}.
+If you are installing fapolicyd along with the new {SmartProxyServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {SmartProxy} rules automatically.
+endif::[]
+
+.Prerequisite
+* Ensure your host has access to the base repositories of {EL}.
+
+.Procedure
+. Install fapolicyd:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+ifndef::foreman-deb[]
+# dnf install fapolicyd
```

Comment:
Why do you use `dnf` here on EL? IMHO the attribute also applies for katello etc: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L88C2-L88C25

````
$ rg ":package-install-project:"
guides/common/attributes-foreman-deb.adoc
10::package-install-project: apt-get install

guides/common/attributes-satellite.adoc
86::package-install-project: satellite-maintain packages install

guides/common/attributes-base.adoc
88::package-install-project: dnf install
````

Reply:
`package-manager` does not work with `Katello` and `Orcharhino`.
`package-install-project` has `satellite-maintain` instead of `dnf` for `Satellite`. 
`package-manager` does not work for Satellite in `Disconnected` mode but works with `Connected` one (strange).

The only option I had was this. :cry: 
An issue worth fixing.

Reply:
Yes. I will open :+1:

Reply:
https://github.com/theforeman/foreman-documentation/issues/2621

CC: @maximiliankolb